pkg/ovsdb/dbserver.go

```diff
@@ -19,14 +19,25 @@ import (
 	"github.com/ibm/ovsdb-etcd/pkg/types/_Server"
 )
 
+type DBServerInterface interface {
```

Comment:
I would suggest separating OVSDB methods, such as `Lock`, `Unlock`, GetSchema`, GetUUID` and others

Reply:
I rather this be done in future, so now I just wanted to make the *existing* code testable